diff -Naur orig/Makefile new/Makefile
--- orig/Makefile	2012-06-13 16:14:20.000000000 +0200
+++ new/Makefile	2020-09-25 23:37:27.590344599 +0200
@@ -7,7 +7,6 @@
 	mkdir -p $(prefix)/include/libspotify
 	install include/libspotify/api.h $(prefix)/include/libspotify
 
-	mkdir -p $(prefix)/lib
 	install -T lib/libspotify.so.12 $(prefix)/lib/libspotify.so.12.1.51
 
 	cd $(prefix)/lib && ln -sf libspotify.so.12.1.51 libspotify.so.12 && ln -sf libspotify.so.12.1.51 libspotify.so
@@ -15,7 +14,6 @@
 	mkdir -p $(prefix)/lib/pkgconfig
 	sed -e s:PKG_PREFIX:$(prefix):g <lib/pkgconfig/libspotify.pc >$(prefix)/lib/pkgconfig/libspotify.pc
 
-	ldconfig
 
 uninstall:
 	rm -rf $(prefix)/include/libspotify
